ci-operator/config/openshift/ironic-rhcos-downloader/openshift-ironic-rhcos-downloader-release-4.2.yaml

```diff
@@ -0,0 +1,35 @@
+base_images:
+  base:
+    name: "4.2"
+    namespace: ocp
+    tag: base
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
+    name: release
+    namespace: openshift
+    tag: golang-1.10
+images:
+- dockerfile_path: Dockerfile.ocp
+  from: base-os
+  to: ironic-rhcos-downloader
+promotion:
+  disabled: true
+  name: "4.2"
+  namespace: ocp
+raw_steps:
+- pipeline_image_cache_step:
+    commands: |
+      curl http://base-openstack-4-2.ocp.svc > /etc/yum.repos.d/base-openstack-4-2.repo
```

Comment:
What do we need from openstack for this image?  It looks like we just install `qemu-img` and `jq` from rpms?

Reply:
This was a wrong copy-paste, I'll remove it